packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -53,21 +53,50 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
 
   public render(): JSX.Element {
     this._adjustProps();
-    const { data, hideNumberDisplay, hideLegend, theme, styles, barBackgroundColor, href, ignoreFixStyle, hideDenominator } = this.props;
+    const {
+      data,
+      benchmarkData,
+      targetData,
+      hideNumberDisplay,
+      hideLegend,
+      theme,
+      styles,
+      barBackgroundColor,
+      href,
+      ignoreFixStyle,
+      hideDenominator
+    } = this.props;
     const { palette } = theme!;
     const barHeight = ignoreFixStyle || data!.chartData!.length > 2 ? this.props.barHeight : 8;
-    const bars = this._createBarsAndLegends(data!, barHeight!, palette, barBackgroundColor, href);
+    if (benchmarkData) {
+      // benchmark color is used to render color for benchmark triangle and benchmark legend
+      benchmarkData.color = benchmarkData.color || palette.neutralTertiary;
```

Comment:
I know your point. But benchmark color is used to render color for both **benchmark triangle** and **benchmark legend**. So I think it is necessary to set a default color in benchmarkData.

If we move this bit into styles, it just set the default color for benchmark triangle, we still need to set a default color for benchmark legend which should be hardcode in StackedBarChart.base.tsx. It's hard to keep the colors the same because consumer can override the .triangle style to change the color of benchmark triangle, but it won't change the color of benchmark legend. This causes the triangle and the legend color to be inconsistent. 

If consumer want to change the color of benchmark, they can pass the customized color in benchmarkData.